### ISSUE TITLE
Dialog disappear on device rotation

### ISSUE THREAD

**Issue by reverse-unina** [SimpleMobileTools/Simple-Gallery#88]
When i perform these sequences of events, the dialog on screen disappears:

Sequence 1:
1. Click on Sort icon
2. Orientation Change

Sequence 2:
1. Long click on item in gallery
2. Click on Info icon
3. Orientation Change

Sequences 3-4-5:
1. Long click on item in gallery
2. Click on ... to open action overflow menu
3. Click on:
a. Rename
b. Copy/Move
c. Delete
4. Orientation Change

Sequence 6:
1. Long click on item in gallery
2. Click on ... to open action overflow menu
3. Click on Copy/Move
4. Click on Select Destination
5. Orientation Change

Sequence 7:
1. Click on Gallery folder item
2. Click on Sort files icon
3. Orientation Change

Sequences 8-9:
1. Click on Gallery folder item
2. Long click on file item
3. Click on ... to open action overflow menu
4. Click on:
a. Copy/Move
b. Delete
5. Orientation Change

**Comment by trubitsyn** [comment]
@tibbi does not reproduce with latest version.

**Comment by tibbi** [comment]
none of them?

**Comment by trubitsyn** [comment]
None of them. Though here's another issue: the Sequence 6 raises an exception if I trigger an orientation change once again.

**Comment by tibbi** [comment]
ok, thanks. Ill leave it here then, tho I dont want to spend time at optimizing device rotations now.

**Comment by BraINstinct0** [comment]
On current latest(6.22.0), after selecting the three dots, rotation simply cancel(exit) the menu.
tested on Android 11

None other creates any crash or exception, dunno when this was solved but now it seems to be.

**Comment by BraINstinct0** [comment]
Edit: actually, this might be due to #89 .
Should be tested after #89 is addressed.
